### ISSUE TITLE
[va-alert]: Alert heading partially announced on iOS

### ISSUE THREAD

**Issue by darbyemadewell** [department-of-veterans-affairs/vets-design-system-documentation#5499]
### Point of contact

@darbyemadewell

### How severe do you think this issue is?

Medium

### Severity rationale

When navigating by heading on iOS, alert headings are only partially announced (the alert level and heading level, not the content). However, the heading content is available by navigating to the next element. Screen reader users may navigate to the next element after the alert is announced to get more information, but it would be preferred for the entire heading to be announced at once.

If multiple alerts are present on the same page, the same heading may be announced multiple times and could cause confusion.

### URL where issue occurs

https://design.va.gov/storybook/?path=/docs/uswds-va-alert--docs

### Component or pattern (if applicable)

va-alert

### Environment details

- OS: iOS 26.2
- Browser: Safari
- Screen Reader: VoiceOver
- Device: iPhone 12 Pro

### Steps to reproduce

- Navigate to https://design.va.gov/storybook/?path=/docs/uswds-va-alert--docs
- Use the VO rotor to navigate by heading
- Swipe through the headings on the page
- Notice that each alert heading is announced as "[alert level] alert, heading level [heading level]" (example: "Information alert, heading level 2")
- Notice that the heading following heading text is not announced unless you swipe left and right to navigate between elements

### Observed behavior

When navigating by heading using iOS, each alert heading is announced as its alert level and heading level. The following heading text is not announced.

### Expected behavior / Acceptance criteria

This behavior is expected for VoiceOver on iOS because it only announces the first element with in the heading (the `span`, in this case). However, it would be preferred for the entire heading to be announced at once.

### Supporting evidence (screenshots, recordings, code snippets)

https://github.com/user-attachments/assets/8c67209c-01aa-4b4b-be4f-1b010d38ea84

### Type of issue

- [ ] Axe-core
- [ ] Cognitive | user flows & navigating
- [ ] Color, typography, & visual elements
- [ ] Components and pattern standards
- [ ] Content organization
- [ ] Content style guide
- [ ] Focus
- [ ] Headings
- [ ] Keyboard
- [ ] Link and button labels
- [ ] Markup and metadata
- [ ] Non-text content and media
- [ ] Overall user experience
- [x] Screenreader | assistive tech & device support
- [ ] Semantic HTML
- [ ] URL standards
- [ ] Zoom or Magnification
- [ ] Something else
